permissions/api_permissions/graphene_permissions.py

```diff
@@ -36,6 +38,56 @@ def has_mutation_permission(cls, root: Any, info: ResolveInfo, input: dict) -> b
         return False
 
 
+class ApplicationPermission(BasePermission):
+    @classmethod
+    def has_permission(cls, info: ResolveInfo) -> bool:
+        return info.context.user.is_authenticated
```

Comment:
This is quite a ponder actually... I would say only staff users with proper permissions would be able to view applications. At least some of the details. But can't tell for certain now....

Reply:
Hmm... The permission is modeled after the REST API permission. Also, the "main" filtering of which applications are included in the "applications" query, is defined in the `ApplicationsFilter` `resolve_queryset` method.

But, I do think it would be good to "doublecheck" that a user has permission to access some application! So, the permission can be improved to include more elaborate checking.

Reply:
yes, there might be some information that does not concern the public viewer.

Reply:
I don't  know if this is what you meant, but I made the permission tests more robust. Unit-, unit group- and service sector permissions now create a new type of application that the tested user should not have access to. This should make tests more robust that users do not have access to applications they should not see.

I also added "regular user" test to the permissions, even though this is kind of tested in the feature tests. But, just to make sure that permissions/ filtering is working.